src/Containers/Microsoft.NET.Build.Containers/LocalDaemons/DockerCli.cs

```diff
@@ -276,18 +285,51 @@ private static async Task WriteImageToStreamAsync(BuiltImage image, SourceImageR
         // Try to find the docker or podman cli.
         // On systems with podman it's not uncommon for docker to be an alias to podman.
         // We try to find podman first so we can identify those systems to be using podman.
```

Comment:
Nit: it's not "first" any more.

Reply:
tweaked this comment to align with the reality of the method now